### DISCUSSION TITLE
Wechsel/Anpassung des Balkonkraftwerks (HMS-2000)

### DISCUSSION THREAD

**Discussion by ckuntze** [hoylabs/OpenDTU-OnBattery#1139] [General]
Hallo,

habe seit kurzem ein Hoymiles HMS-2000 mit 4 Pannels mit 450W.
String 1 - 1 Panel
String 2 - 1 Panel
String 3 - 1 Panel
String 4 - 1 Panel

Da ich nun ein wenig angefixt bin würde ich es mit einer Batterie erweitern....
Daher hätte ich ein paar Fragen am euch ob alles sowei richtig verstanden habe und es so Funktionieren würde

Step 1
String 1 - 2 Panel in Reihe über Victron MPPT 150/35 (inkl. Leistungsschutzschalter 50A)
String 2 - Paralell mit String 1 Anschliessen
String 3 - 2 Panel in Reihe über Victron MPPT 150/35 (inkl. Leistungsschutzschalter 50A)
String 4 - Paralell mit String 3 Anschliessen

Step 2
String 1 - 2 Panel in Reihe über Victron MPPT 150/35
String 2 - Paralell mit String 1 Anschliessen
String 3 - 2 Panel in Reihe über Victron MPPT 150/35 / Pylontech US3000 oder US5000
String 4 - Paralell mit String 3 Anschliessen

Wäre für eure Anmerkungen Dankbar

Gruß
Carsten

**Comment by schlimmchen** [comment]
Also ich verstehe nicht, was du vor hast. Jedenfalls ergibt es keinen Sinn für mich. Vielleicht magst du mal ein Diagramm malen? Allgemein gesprochen: Alle Solarmodulen sind an der PV-Seite der Laderegler angeschlossen, deren Batterie-Seite an der Batterie angeschlossen ist. Der Wechselrichter wird an der Batterie angeschlossen.

**Reply by ckuntze** [reply]
Hallo,

habe mal auf die schnelle was Zusammengezimmert....

Oben ist der Iststand
Mitte ist Step1
Unten wäre Step 2

Ich hoffe das das so Verständlicher ist

![Unbenannt](https://github.com/user-attachments/assets/21ae6cc6-ddea-4405-8aca-cde926d3de19)

**Reply by AndreasBoehm** [reply]
Step 1 musst du überspringen, die Schwankungen die durch die 4 MPPT deines Wechselrichtrts ausgelöst werden können die Victrons nicht ab, da wird nicht viel Leistung rumkommen.
Probiert hab ich das noch nicht aber das liest man überall in Erfahrungsberichten.

Direkt dir Batterie kaufen ist die einzig sinnvolle Option.

**Reply by spcqike** [reply]
Step 1 macht auch gar keinen Sinn :) warum sollte man die zwei Paneele als String betreiben und dann parallel am HMS anschließen, mit allen Nachteilen die ein String mit sich bringt, wenn der HMS sie beide direkt einzeln am MPP betreiben könnte?

**Reply by ckuntze** [reply]
Moin,

Sinn wäre nicht gleich soviwl Geld auszugeben....
Dann wird es wohl gleich das volle paket.

Danke euch

**Reply by spcqike** [reply]
Naja. Jetzt zwei laderegler und passende Kabel kaufen, und dann weniger Effizienz haben? 

Dann warte lieber bis du alles Geld beisammen hast und Kauf ggf die laderegler wenn bis dahin n gutes Angebot kommt.

**Reply by AndreasBoehm** [reply]
Ich bin mir auch nicht sicher ob du bei 4 Modulen mehr als einen Victron benötigst, ich denke das 1x 150/35 ausreichen sollte, maximal 1x 150/45. 

Hast du den Victron Rechner benutzt?

**Reply by spcqike** [reply]
Da hat @AndreasBoehm sehr wahrscheinlich recht. Ich denke auch, dass ein 150/35 ausreicht. Die Paneele kommen dann als 2s2p angeschlossen.

**Reply by ckuntze** [reply]
Mahlzeit

Ja, den hatte ich benutzt und bin auch zu dem Schluss gekommen.... das es eher ein 150/35 wird...
Macht die Verkabellung auch etwas Überschaubarer

**Reply by schlimmchen** [reply]
Und dann werden auch alle Eingänge des Inverters aus der Batterie gespeist (bzw. von ihr gestützt)? Ein Mischbetrieb wie im unteren Teil deiner Skizze (ein Teil der Eingänge an der Batterie, ein anderer nicht (egal ob am Laderegler oder an Solarmodulen)) wird *nicht* unterstützt.

**Comment by Manos1966** [comment]
@ckuntze 

vielleicht hilft dir:
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki
und
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Builds-%26-Examples

**Reply by ckuntze** [reply]
Das hatte ich soweit schon durchgearbeitet.
Aber bei den vielen Möglichkeiten wollte ich nur Wissen ob mein Ansatz richtig ist.

**Reply by Manos1966** [reply]
Ah so! 
(👍 👍 👍 , dass du das Wiki vorher probiert hast).

> Oben ist der Iststand
Mitte ist Step1
Unten wäre Step 2

Dann, wuerde ich (auf Basis der Bilder), sagen:

IstStand: ist OK

Step1: um Gottes Willen! Warum? Die Victron MPPTs sind gemacht um Batterien zu laden, nicht um Wechselrichter zu versorgen = unkalkulierbares Risiko, MPPT-Tracker auf Victron Seite, MPPT-Tracker auf HMS-Seite. 

Step2: Der rechte Victron der an der Batterie angeschlossen ist= perfekt
Verbesserungs Moeglichkeit: Allee vier Strings, also String1+2+3+4 mit der Batterie anschließen. Muss nicht, koennte es aber= haengt davon ab, wieviel Watt deine HMS liefern soll= Jeder HMS String kann (glaube ich) max 15A, daher 48V Batterie x 15A = 720W max Leistung pro String. HMS Effizienz ist 96% also 691W AC-Seite.  Du hast nur zwei angeschlossen, also kann dein HMS  bis zu 1382W AC-Seitig liefern. Vielleicht reicht es fuer deine Zwecke aus.
Haettest du mehr Leistung gebraucht, muesstest du die anderen Strings auch anschliessen, oder vielleicht alle Strings direkt anschliessen, damit um eine bessere Last-Verteilung auf alle Strings zu erreichen.
Der linke Victron, nicht an der Batterie angeschlossen, gleiches Kommentar wie Step1



PS. Alles ist meine persönliche Meinung, ich bin kein Spezialist 🙊

**Reply by ckuntze** [reply]
Ja, der Plan bei mir hat sich mittlerweile auch schon etwas geändert....
Werden wohl alle 4 Platten über einen Victron 250/60 laufen....

Danke

**Reply by spcqike** [reply]
@ckuntze warum genau nimmst du den 250/60? ist der mit 3440W 48V Ladeleistung nicht etwas überdimensioniert für 4 Paneele? tut es nicht ein 150/35 mit immer noch 2000W Ladeleistung?`Nur, falls du das Ding noch nicht gekauft hast, die PV nicht wesentlich erweitern willst und Geld sparen willst :) 300€ oder 130€, ist schon nicht unwesentlich.

**Reply by ckuntze** [reply]
Aufgrund des etwas ungünstigen Kabelweges…
Es gibt zwar noch den Gedanken zwei in Reihe und dann parallel zu machen, dann brauche ich aber andere kabel (habe schon was bekommen)

**Reply by spcqike** [reply]
Warum nicht nur einen 150/35 und die Paneele in 2s2p anschließen? Für 170€ die du bei dem laderegler sparst kannst du allerhand Kabel kaufen.

**Reply by Sorbat** [reply]
> die Paneele in 2s2p anschließen? 

geht das so ohne weiteres? wie wäre da der Aufbau? alles was von den Panels kommt auf eine Busbar und von der Busbar zum MPPT?

**Reply by AndreasBoehm** [reply]
@Sorbat Busbar klingt nach mehr als das was man braucht. Kannst du auch mit Reihenklemmen machen.
Zusammengeschlossen wir das ganze dann wie hier (Bild aus dem wiki): ![img](https://private-user-images.githubusercontent.com/73672554/263498863-645b7a4d-8ea1-419b-aa7c-3c17fc4c2fb2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI5NjQ0MDYsIm5iZiI6MTcyMjk2NDEwNiwicGF0aCI6Ii83MzY3MjU1NC8yNjM0OTg4NjMtNjQ1YjdhNGQtOGVhMS00MTliLWFhN2MtM2MxN2ZjNGMyZmIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODA2VDE3MDgyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQzNjJmZDNiMjU0MzUwMzU0MWRkYWY1MGExODE5Y2Q1ODcyNzNjMGVkYzNiYTc4ZGRhZDdlNzkxODgzODBiYTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ENP4ROWs5KlBNXH0_4ojMDId0hMQoj80JwXJvkSbzDU)

**Reply by spcqike** [reply]
Oder mit Y-Adaptern. 

Aktuell habe ich am HMS-2000 auch jeweils 1s2p angeschlossen. Mehr Strom wird es ja bei 2s2p auch nicht. Nur mehr Spannung.

**Reply by AndreasBoehm** [reply]
Das hängt aber sehr stark von Modulen ab ob der MC4 Stecker den Kurzschlusstrom tatsächlich verträgt.

**Reply by spcqike** [reply]
Gibt es Module die > 15A liefern? Der MC4 Stecker hat 30A (Dauer)Belastbarkeit, oder?

**Reply by ckuntze** [reply]
Natürlich darf in der Konstellation die Verkabelung nicht außer Acht gelassen werden.

Derzeit sind es 4 Platten (450w spitze bei ca 40v) am hoymiles direkt, kurze Kabel, hoymiles direkt bei den Panels.
Wenn der Umbau erfolgt, Akku, ladereglerund hoymiles innen, habe ich halt ca 15-20m Strecke - da ist schon die Frage 2s2p oder 4s1p schon zu berücksichtigen

**Reply by spcqike** [reply]
> habe ich halt ca 15-20m Strecke - da ist schon die Frage 2s2p oder 4s1p schon zu berücksichtigen

2s2p kostet dich halt ~30m mehr an Solarkabel und entsprechende MC4 oder Klemmverbindungen.

Auch wenn es ginge, die Strings schon auf dem Dach parallel zu schalten und nur eine Leitung ins Haus zu führen, würde ich dennoch 2 Leitungen runter führen. Das macht vieles einfacher und wahrscheinlich sicherer. und du kannst deine Strings, solltest du es benötigen, nachträglich trennen und untersuchen (Spannung oder Strom messen, wenn du Zweifel hast, ob noch alle Paneele korrekt funktionieren) ohne aufs Dach steigen zu müssen.

ich hab von jedem Paneel einzelne Kabel in die Garage geführt, also 16x 10m 6qmm Kabel. So kann ich es später auch entspannt umklemmen, wenn ich will.

**Reply by DonJohnLong** [reply]
> Natürlich darf in der Konstellation die Verkabelung nicht außer Acht gelassen werden.
> 
> Derzeit sind es 4 Platten (450w spitze bei ca 40v) am hoymiles direkt, kurze Kabel, hoymiles direkt bei den Panels. Wenn der Umbau erfolgt, Akku, ladereglerund hoymiles innen, habe ich halt ca 15-20m Strecke - da ist schon die Frage 2s2p oder 4s1p schon zu berücksichtigen

Ich habe 6x 400w an einem 150/45 in 3s2p und ist mit ca. 20m kabel völlig unproblematisch.

Lg

**Reply by Manos1966** [reply]
> habe ich halt ca 15-20m Strecke - da ist schon die Frage 2s2p oder 4s1p schon zu berücksichtigen

Wahrscheinlich hast du dich bereits schlau gemacht wenn du Frage so stellts...
dann ist es nur eine Wiederholung:
Dickerer Kabel Querschnitt = weniger Verluste
Hoehere Spannung (V) = weniger Verluste
Hoehere Strom (A) = hoehere Verluste

So gesehen, waere eine 4S1P besser.
Der Preis-Unterschied ist wir Sorbat erwaehnt hat, das doppelte (130EUR vs 300EUR fuer den MPPT250).
FALLS du vor hast, eines Tages mehr als 4 Paneele zu anschliessen, dann ist der MPPT250 interessant.
FALLS du grosse Batterien hast (die im Winter geladen muessen), dann ist der MPPT250 mit seiner 60A interessant (anstatt 35A des MPPT150).
FALLS du erst "testen" willst, dann sind die 130EUR des MPPT 150/35 eine gute "Start-Investition". Vorsicht bei der Planung in diesem Fall, der MPPT250 ist viel grosser als der MPPT150...

**Reply by AndreasBoehm** [reply]
> Gibt es Module die > 15A liefern? Der MC4 Stecker hat 30A (Dauer)Belastbarkeit, oder?

Du hast recht, ich hab mich verrechnet. 2s2p ist unbedenklich an einem MC4 Stecker.

**Reply by spcqike** [reply]
> FALLS du grosse Batterien hast (die im Winter geladen muessen), dann ist der MPPT250 mit seiner 60A interessant (anstatt 35A des MPPT150).

hm. Liefern nicht beide Modelle mehr ladeleistung als die 4 bisher vorhandenen Paneele liefern können?

der 250/60 lohnt sich meiner Meinung nach nur, wenn man zukünftig weitere Paneele anschließen will und halt mehr als 2.000W effektiver ladeleistung bekommt. 

4x450W könnten zwar 1800W liefern, aber wann schaffen wir das in Deutschland? SCT Laborwerte hin oder her, NOCT ist halt langfristig realistischer. Und da kommen wir halt eher auf ~350W pro Modul (je nach Modul, Effizienz, Temperatur,….. ihr versteht mich :D )

**Reply by Sorbat** [reply]
Ich hatte mal geplant 4 (2s2p) Panels auf die eine Seite zu machen, 2 Panels auf die andere Seite. Ich hatte einen 150/45 gekauft und wollte mal schauen, wie weit ich damit komme. Wie sich dann herausstellen sollte, passen aber da wo die 4 geplant waren nur 3 hin, daher hatte ich mir dann noch einen 250/70 gekauft (weil 3x60v >150). Den 150er kann ich dann ja für die 2 auf deren Seite nutzen. 
Aber so wie ich jetzt lese, hätte ich für die 3 Panels statt dem 250er auch die 3 parallel an den kleinen mppt hängen können?

**Reply by spcqike** [reply]
>  hätte ich für die 3 Panels statt dem 250er auch die 3 parallel an den kleinen mppt hängen können?

kommt auf die MPP Spannungen und deine Batteriespannung an. die Module müssen unter Last etwas mehr Spannung liefern, als deine Batterie braucht.

aus dem Datenblatt des 150/35:
![grafik](https://github.com/user-attachments/assets/2d0f528d-f9eb-4373-9072-ebb8b9c56f2f)

**Reply by Sorbat** [reply]
Ah okay, ja dann war wohl das das Problem 😄

**Comment by spcqike** [comment]
@Sorbat 
> geht das so ohne weiteres?

ja, sollte. es gibt natürlich viele Faktoren, die so ein Setup beeinflussen und es uneffizienter machen, als 2 einzelne Laderegler sein könnten. (masgeblich (teil-)Verschattung, die Ausrichtung der Module), aber wirtschaftlicher ist in der Regel 1 Laderegler. So viel mehr kWh muss man erstmal erzeugen, als dass sich die Mehrkosten rechnen.

siehe dazu gern auch Diskussion #249 und den dort verlinkten externen Beitrag.
